crates/matrix-sdk/tests/integration/room/common.rs

```diff
@@ -132,6 +132,54 @@ async fn test_room_names() {
         ),
         room.compute_display_name().await.unwrap()
     );
+
+    // Room with joined and invited members.
+    let room_id = room_id!("!joined_invited_members:localhost");
+    sync_builder.add_left_room(LeftRoomBuilder::new(room_id).add_state_bulk([
+        sync_state_event!({
+            "content": {
+                "membership": "join",
+            },
+            "event_id": "$example1_join",
+            "origin_server_ts": 151800140,
+            "sender": "@example1:localhost",
+            "state_key": "@example1:localhost",
+            "type": "m.room.member",
+        }),
+        sync_state_event!({
+            "content": {
+                "displayname": "Example 2",
```

Comment:
maybe name `Alice` or `Bob` to make clear it's a displayname and not a user id localpart?

Reply:
Done